website/cue/reference/remap/functions/set_event_metadata.cue

```diff
@@ -0,0 +1,39 @@
+package metadata
+
+remap: functions: set_event_metadata: {
+	category: "System"
+	description: """
+		Sets the given field in the event metadata to the provided value.
+		"""
+
+	arguments: [
+		{
+			name:        "key"
+			description: "The name of the key to set in the metadata."
+			required:    true
+			enum: {
+				"datadog_api_key": "The Datadog api key."
```

Comment:
Could we add a note here that this will be used by the `datadog_*` sinks as the API key to send the events with?